src/components/sdkApi.tsx

```diff
@@ -0,0 +1,158 @@
+import {Fragment} from 'react';
+
+import {getCurrentPlatform} from 'sentry-docs/docTree';
+import {serverContext} from 'sentry-docs/serverContext';
+import {PlatformCategory} from 'sentry-docs/types';
+
+import {Expandable} from './expandable';
+import {codeToJsx} from './highlightCode';
+import {SdkDefinition} from './sdkDefinition';
+import {getPlatformHints} from './sdkOption';
+
+export interface ParameterDef {
+  name: string;
+  type: string | ObjectParameterDef;
+  defaultValue?: string;
+  description?: string;
+  required?: boolean;
+}
+
+type ObjectParameterDef = {
+  properties: ParameterDef[];
+  name?: string;
+};
+
+type Props = {
+  name: string;
+  parameters: ParameterDef[];
+  signature: string;
+  categorySupported?: PlatformCategory[];
+  children?: React.ReactNode;
+  language?: string;
+};
+
+export function SdkApi({
```

Comment:
I think it is super duper critical that we make the content of the API searchable (cmd+f) even if the expandibles are collapsed. Otherwise I would maybe argue it is better to expand them by default.